test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/math/BigDecimalArithmeticTest.scala

```diff
@@ -1405,4 +1405,11 @@ class  BigDecimalArithmeticTest {
     assertEquals(result.toString, c)
     assertEquals(result.scale(), cScale)
   }
+
+  // https://github.com/scala-js/scala-js/issues/2587
+  @Test def testZeroPow0(): Unit = {
```

Comment:
The method name does not seem to fit. It seems you copy-pasted the method just above but forgot to change the name ;)